test/unit-test/mqtt_agent_utest.c

```diff
@@ -1355,7 +1417,7 @@ void test_MQTTAgent_CommandLoop_with_eventCallback( void )
 
     TEST_ASSERT_EQUAL( MQTTSuccess, mqttStatus );
     /* Ensure that callback is invoked. */
-    TEST_ASSERT_EQUAL( 6, commandCompleteCallbackCount );
+    TEST_ASSERT_EQUAL( 7, commandCompleteCallbackCount );
```

Comment:
We should write our tests such that rearranging some or inserting one in the middle shouldn't require incrementing a variable in every assert that follows. If they're testing separate things, they should be separate tests, or this variable should be reset between API calls.